docs-md/getting-started/dependencies.md

```diff
@@ -35,6 +35,6 @@ Capacitor uses the WKWebView.
 
 Android development requires the **Android SDK** installed with **[Android Studio](https://developer.android.com/studio/index.html)**. Technically, Android Studio isn't required as you can build and run apps using only the Android CLI tools, but it will make building and running your app much easier so we strongly recommend using it.
 
-Android version support for Capacitor is more complex than iOS. Currently, we are targeting API level 21 or greater, meaning Android 5.0 (Lollipop) or above. [As of May 2019](https://developer.android.com/about/dashboards), this represents over 89% of the Android market.
+Android version support for Capacitor is more complex than iOS. Currently, we are targeting API level 21 or greater, meaning Android 5.0 (Lollipop) or above. This represents over 90% of the Android market.
```

Comment:
Took this out since it gets out of date plus it's no longer possible to see updated stats (Google claims it'll be back but it's been many months without an update). So, I think the updated sentence works for explaining that Capacitor supports most users.